### ISSUE TITLE
Adding Delivery Calendar

### ISSUE THREAD

**Issue by ashokrane** [TycheSoftwares/order-delivery-date-for-woocommerce#439]
I am thinking we should add the Delivery Calendar in the Lite version too. That's the one page where store owners will visit every day to view their orders.

Along with that, we can add a story/case study on the same Delivery Calendar page. So everytime a store owner goes to the Delivery calendar page, they will see the case study description & be prompted to upgrade to Pro. We have short story forms of case studies written [here](https://docs.google.com/document/d/1S6Hu4GoGTAu4vxitTITcZ6-xEPmFDaTE4KqHMPM5c3Q/edit?usp=sharing).

What are your thoughts @rashmimalpande @Chetna1510 @upenkapse?

**Comment by upenkapse** [comment]
I agree @ashokrane, Delivery calendar will be a very good addition to the Lite version, and as you mentioned we can add a case study brief as a quote on that page with a CTA button to upgrade to Pro.

**Comment by rashmimalpande** [comment]
@ashokrane yes, this sounds good. Delivery Calendar would be useful to store owners and adding a story instead of just some text to upgrade is a good idea.
